Refresh.GameServer/RefreshGameServer.cs

```diff
@@ -255,7 +224,7 @@ public void SetAdminFromEmailAddress(string emailAddress)
         context.SetUserRole(user, GameUserRole.Admin);
     }
 
-    public void Dispose()
+    public override void Dispose()
     {
         this.Logger.Dispose();
         this._databaseProvider.Dispose();
```

Comment:
Some of this code is duplicated from the base implementation, you can just do `base.Dispose()` instead of duplicating the `Logger.Dispose` and `GC.SuppressFinalize`

Reply:
True, didn't realize I had duplicated this